### PR TITLE
Exact stock level shouldn't be exposedto the end client

### PR DESCRIPTION
**Pull request by artursmet**
As we discussed, front-end shouldn't be aware of exact stock level for particular product.
This is work in progress - cart quantity widget should be upated and it shouldn't allow user to submit a form when quantity is wrong.

### CODE REVIEW

**Comment by artursmet** [comment]
@benekex2 could you take a look and update cart line quantity widget on cart details view?

**Comment by patrys** [comment]
I think we're sacrificing user experience for the sake of faux secrecy. As a customer I can still trivially determine the exact amount left in stock by trying to add increasingly lower number.

File: saleor/settings.py [hunk 1]
```diff
@@ -258,6 +258,7 @@
     messages.ERROR: 'danger'}
 
 LOW_STOCK_THRESHOLD = 10
+MAX_CART_LINE_QUANTITY = os.environ.get('MAX_CART_LINE_QUANTITY', 50)
```
Comment (patrys) [inline saleor/settings.py]: Missing docs 😉

**Comment by artursmet** [comment]
@patrys Docs updated.

**Pull request merged**